vdr/vdr.go

```diff
@@ -132,7 +132,10 @@ func (r *VDR) Diagnostics() []core.DiagnosticResult {
 		totalCount++
 		controllers, err := r.didDocResolver.ResolveControllers(doc, &types.ResolveMetadata{Hash: &metadata.Hash})
 		if err != nil {
-			log.Logger().Errorf("failed to resolve controller for %s: %v", doc.ID, err)
+			log.Logger().
```

Comment:
Revert this, change not part of PR

Reply:
`go vet` was complaining about this and I was unable to run my tests. But I cannot trigger this problem again.